src/Typechecker.ml

```diff
@@ -18,6 +18,20 @@ let with_error_location loc r =
 
 let fail_in loc err = Result.fail @@ Location.locate ~loc @@ err
 
+(** [mk_unbound_regular_var_inside_box_error loc var_name] create
+    [`UnboundRegularVarInsideBoxError] with human readable message.
+ *)
+let mk_unbound_regular_var_inside_box_error box_expr_loc var_loc var_name =
+  (* let var_loc_msg = Location.pp_column_range var_loc in
+   * let box_expr_loc_msg = Locations.pp_column_range box_expr_loc in *)
```

Comment:
I guess this commentary is not needed anymore :)
```suggestion
```

Reply:
fixed